<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plot</class>
 <widget class="QDialog" name="plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>685</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>685</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X min:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X max:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y min:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>135</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y max:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_xMin">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>15</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_xMax">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>55</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_yMin">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>95</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_yMax">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>135</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_print">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>170</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  margin: 0 0 5px;
  padding: 0 12px;
  font-size: 16px;
  font-weight: bold;
  color: #eeeeee;
  background: #6060cc;
  border-width: 1px 1px 1px;
  border-style: solid;
  border-color:   #cecece #bababa #a8a8a8;
  border-radius: 13px 13px 12px 12px;
}

QPushButton::hover
{
  background: #7e7ed6;
  border-top-color: #c9c9c9;
}

QPushButton::pressed
{
  vertical-align: -5px;
  background: #313199;
  margin-bottom: 0;
  padding: 1px 13px 0;
  border-width: 0;
  border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>210</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono NL</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  margin: 0 0 5px;
  padding: 0 12px;
  font-size: 16px;
  font-weight: bold;
  color: #eeeeee;
  background: #c24242;
  border-width: 1px 1px 1px;
  border-style: solid;
  border-color:   #cecece #bababa #a8a8a8;
  border-radius: 13px 13px 12px 12px;
}

QPushButton::hover
{
  background: #cc6060;
  border-top-color: #c9c9c9;
}

QPushButton::pressed
{
  vertical-align: -5px;
  background: #993131;
  margin-bottom: 0;
  padding: 1px 13px 0;
  border-width: 0;
  border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>500</width>
     <height>490</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>490</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>490</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
